companion/discord_bot/bot.py

```
@@ -0,0 +1,435 @@
+                except:
+                    pass
+
+            # Acquire user lock and process message
+            lock_acquired = False
+            try:
+                async with lock:
+                    lock_acquired = True
+                    try:
+                        # Process the message via gateway API
+                        await self._process_message_via_gateway(message)
+                    except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)